app/api/tax.py

```diff
@@ -43,9 +65,38 @@ class Meta:
                          type_='event')
 
 
-class TaxList(ResourceList):
+class TaxListPost(ResourceList):
     def before_post(self, args, kwargs, data):
         require_relationship(['event'], data)
+        if not has_access('is_coorganizer', event_id=data['event']):
+            raise ForbiddenException({'source': ''}, 'Co-organizer access is required.')
+
+    def before_get(self, args, kwargs):
+        if has_access('is_admin'):
+            self.schema = TaxSchema
+        else:
+            self.schema = TaxSchemaPublic
+
+    schema = TaxSchema
+    data_layer = {'session': db.session,
+                  'model': Tax}
+
+
+class TaxList(ResourceList):
+    def before_get(self, args, view_kwargs):
+        if view_kwargs.get('event_identifier'):
+            try:
```

Comment:
So if session is not booting up here, Can before_get_collection method be used here? (just curious)